ext/DifferentiationInterfaceFiniteDifferencesExt/DifferentiationInterfaceFiniteDifferencesExt.jl

```diff
@@ -19,4 +19,15 @@ function DI.value_and_pushforward(
     return y, jvp(backend.fdm, f, (x, dx))
 end
 
+#=
+# TODO: why does this fail?
```

Comment:
@adrhill this is the function we need for NNLib, so I'd be interested in the answer

Reply:
How/why does it fail?